instrumentation-genai/opentelemetry-instrumentation-openai-v2/README.rst

```diff
@@ -17,6 +17,53 @@ Installation
 
     pip install opentelemetry-instrumentation-openai-v2
 
+Usage
+-----
+
+Instrumenting all clients
+*************************
+
+When using the instrumentor, all clients will automatically trace OpenAI chat completion operations
+and capture prompts and completions as events.
+
+Make sure to configure OpenTelemetry tracing, logging, and events to capture all telemetry emitted by the instrumentation.
+
+.. code-block:: python
+
+    from opentelemetry.instrumentation.openai_v2 import OpenAIInstrumentor
+
+    OpenAIInstrumentor().instrument()
+
+    client = OpenAI()
+    response = client.chat.completions.create(
+        model="gpt-4o-mini",
+        messages=[
+            {"role": "user", "content": "Write a short poem on open telemetry."},
+        ],
+    )
+
+Enabling message content
+*************************
+
+Message content such as the contents of the prompt, completion, function arguments and return values
+are not captured by default. To enable message content, set the environment variable
+`OTEL_INSTRUMENTATION_GENAI_CAPTURE_MESSAGE_CONTENT` to `true`.
```

Comment:
note: I found that if this is set to false, it currently crashes. Validation if that's only me requested [here](https://github.com/open-telemetry/opentelemetry.io/pull/5575#discussion_r1841594784)

Reply:
thanks for fixing otlp exporter - https://github.com/open-telemetry/opentelemetry-python/pull/4276 !